modules/animationqt/src/sequenceeditor/invalidationsequenceeditor.cpp

```diff
@@ -130,15 +130,15 @@ InvalidationSequenceEditor::InvalidationSequenceEditor(KeyframeSequence& sequenc
     layout->addLayout(subLayout);
     subLayout->setContentsMargins(0, 0, 0, 0);
     subLayout->setSpacing(7);
-    processor_ = new QLineEdit();
-    processor_->setToolTip("Invalidate the specified processor at each network evaluation");
-    subLayout->addWidget(new QLabel("Processor Identifier"), 0, 0);
-    subLayout->addWidget(processor_, 0, 1);
+    path_ = new QLineEdit();
+    path_->setToolTip("Invalidate the specified processor / property at each network evaluation");
```

Comment:
Suggest to add an example as people might not know what a path is/looks like:
(processorId.propertyId, e.g., Canvas or Canvas.saveLayer)